react-swipe.js

```diff
@@ -47,7 +47,8 @@
       disableScroll   : React.PropTypes.bool,
       stopPropagation : React.PropTypes.bool,
       callback        : React.PropTypes.func,
-      transitionEnd   : React.PropTypes.func
+      transitionEnd   : React.PropTypes.func,
+      newSwiper       : React.PropTypes.bool
```

Comment:
@Snorlock I think should be a better name for this. Maybe `reinitSwipeOnUpdate`?